### ISSUE TITLE
React Native Web webpack build error due to enforced PickerIOS import

### ISSUE THREAD

**Issue by jkoutavas** [react-native-picker/picker#300]
I'm on @react-native-picker/picker v1.16.4. I'm getting this error when trying to build (webpack) my React Native app for web:

```sh
ERROR in ./node_modules/react-native-web/src/modules/UnimplementedView/index.js 10:12
Module parse failed: Unexpected token (10:12)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| 
> import type { Node } from 'React';
| 
| import View from '../../exports/View';
 @ ./node_modules/@react-native-picker/picker/js/PickerIOS.web.js 1:261-318
 @ ./node_modules/@react-native-picker/picker/js/index.js 1:444-466
 @ ./src/components/DropdownPicker/DropdownPicker.tsx 1:487-525
 @ ./src/components/DropdownPicker/index.ts 1:77-104
 @ ./src/HelloWorld/components/SampleForm.tsx 1:1055-1090
 @ ./src/HelloWorld/screens/SampleScreen.tsx 1:698-733
 @ ./src/navigation/HomeStack.tsx 1:281-327
 @ ./src/HelloWorld/MainNav.tsx 1:276-311
 @ ./src/navigation/RootNav.tsx 1:322-356
 @ ./src/App.tsx 1:1270-1301
 @ ./index.web.js 1:308-328

webpack 5.47.1 compiled with 1 error in 16885 ms
error Command failed with exit code 1.
```

Which makes sense, seeing `PickerIOS.web.js` is a `UnimplementedView` for web.

My component `DropdownPicker` does an import like so:

```ts
import { Picker } from '@react-native-picker/picker';
```

and I see `@react-native-picker/picker/js/index.js` contains this:

```ts
/**
 * @flow
 */

export {default as Picker} from './Picker';
export {default as PickerIOS} from './PickerIOS';
```

The bug in this scenario is the second export shouldn't be there. The package shouldn't assume we want to pull in `PickerIOS`. It's just 'Picker' I want to import and not bring along 'PickerIOS' too. Let me explicitly import 'PickerIOS' only if I need it.

How can I directly get to just `./Picker` from my `DropdownPicker` component?

**Comment by kapildeyyara2** [comment]
i am getting similar issue.. is there any solution to this?? RNweb is getting crashed due to this

**Comment by gfaraj** [comment]
Getting this too... Any resolution?

**Comment by Naturalclar** [comment]
@jkoutavas thanks for raising the issue,
Should be fixed in v2.0.3 :tada:
